aya/src/maps/mod.rs

```diff
@@ -538,6 +757,35 @@ impl TryFrom<u32> for bpf_map_type {
         })
     }
 }
+
+macro_rules! impl_try_from_map_generic_key_or_value {
+    ($($typ:ty),+ $(,)?) => {
+        $(
+            impl From<$typ> for u32 {
```

Comment:
Someone is getting excited with these macros! :P

I think this should be just an internal utility method and not a from impl. Right now you've made it (accidentally) a public API and I don't think we want to offer Map => u32 publicly

Reply:
Lol yeah I go over excited, I forgot this was as simple as adding a simple `aya::Map` Method, and I do like Macros :)

Reply:
Done!